packages/lib-classifier/src/store/DrawingStore.js

```diff
@@ -1,56 +1,175 @@
-import { addDisposer, getRoot, onAction, types } from 'mobx-state-tree'
+import cuid from 'cuid'
+import { addDisposer, getRoot, getType, isValidReference, onAction, types } from 'mobx-state-tree'
 import { autorun } from 'mobx'
 import { Subject } from 'rxjs'
+import { filter, map, skipUntil } from 'rxjs/operators'
+
+import { LineStore, PointStore } from './markings'
```

Comment:
This can probably use
```js
import { * as Marks } from './markings'
```
then enumerate the values of `Marks` when you need them. That way, you won't need to update this line with new marking models.